### ISSUE TITLE
pypy 7.x builds missing from index

### ISSUE THREAD

**Issue by azrdev** [antocuni/pypy-wheels#16]
On the index page https://antocuni.github.io/pypy-wheels/ubuntu/numpy/ the highest versions I see are pypy 6.x builds, but none for pypy 7.x

#12 indicates that these should exist, as do the travis build logs, e.g. https://travis-ci.org/antocuni/pypy-wheels/builds/537536271

**Comment by rth** [comment]
Yes, wheels for pypy 7.x seem to be missing,
```bash
docker run --rm -it pypy:3.6-7.1.1 /bin/bash
$ pypy3 --version
Python 3.6.1 (784b254d6699, Apr 14 2019, 10:22:42)
[PyPy 7.1.1-beta0 with GCC 6.2.0 20160901]
$ pypy3 -m pip install --index-url https://antocuni.github.io/pypy-wheels/ubuntu numpy
Looking in indexes: https://antocuni.github.io/pypy-wheels/ubuntu
Collecting numpy
  ERROR: Could not find a version that satisfies the requirement numpy (from versions: none)
ERROR: No matching distribution found for numpy
```

I'm wondering if something is wrong with wheels naming in TravisCI config, e.g. from the above-linked logs, PyPy 7.1.1 for Py3.6 builds,
```
numpy-1.16.3-pp271-pypy_41-linux_x86_64.whl
```
not sure what `pp271` corresponds to but I would have expected to see `36` in the name somewhere.

**Comment by rth** [comment]
Also `antocuni/pypy-wheels` docker image was last updated 2 years ago. Shouldn't it be more recent if it included pypy 7.0 ?

**Comment by antocuni** [comment]
Hopefully fixed by bf051118cfe5abfc348523b096c025fc0df66481. As you can see from the commit, the problem was that all the pypy3 builds always used the pypy2 binaries, so we never produced any pypy3 wheel at all.

Hopefully, when this [travis build](https://travis-ci.org/antocuni/pypy-wheels/builds/554347012) finishes we will have PyPy3 wheels. Closing the issue, please re-open it if from some reasons things don't work.

**Comment by antocuni** [comment]
> Also [`antocuni/pypy-wheels`](https://hub.docker.com/r/antocuni/pypy-wheels) docker image was last updated 2 years ago. Shouldn't it be more recent if it included pypy 7.0 ?

I don't really know why docker hub says 2 years ago. The image which is currently used to build wheels is tagged as `0c016fb7`, and it's 2 months old:
```
$ docker images pypywheels/ubuntu14.04
REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE
pypywheels/ubuntu14.04   0c016fb7            489a1998346c        2 months ago        1.64GB
```
